Monika After Story/game/script-topics.rpy

```diff
@@ -15749,3 +15749,33 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_environment",
+            category=['society'],
+            prompt="The environment",
+            random=True
+        )
+    )
+ 
+label monika_environment:
+    m 2eub "[player], do you know why I care about the environment?"
+    m 1esa "Well, there are a few reasons."
+    m 3euu "For one, I think it's beautiful."
+    m 1dsd "Vibrant meadows in the spring, chittering forests in the fall..."
+    m 2hub "Breathtaking beaches in the summer, and shimmering snow in the winter."
+    m 7wsu "Sometimes it can be euphoric just to sit down and admire nature, you know?"
+    m 1lut "The environment also sustains life."
```

Comment:
![3](https://user-images.githubusercontent.com/45746485/93007414-26d3b000-f569-11ea-9e40-4c931c152856.png)
Not sure I like this expression, especially the mouth. Suggestion:

![image](https://user-images.githubusercontent.com/45746485/93007435-5b476c00-f569-11ea-8a44-8e7a92505049.png)

Reply:
Yeah, that looks a lot better as well. I'll adjust that one too, thanks for the suggestion!

(One thing I'll note here is that I really wanted to try and use the triangle if possible, because I feel like I hardly see it in-game so I'm wondering if it's a neglected option).

Reply:
She does use it briefly, once, in a very specific situation, so I wouldn't worry about it not being utilized elsewhere :)